### DISCUSSION TITLE
Copilot student

### DISCUSSION THREAD

**Discussion by AchrafEAE** [community/community#140039] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello, 

I activated the copilot free trial, but then I saw you can get it for free if you're a student. So I verified myself as a student. But now it says I will lose access to copilot after my free trial ends. Or will I still have access? Also how long will I have access to copilot as a student?

**Comment by t1noo7** [comment]
Once you have completed the verification that you are a student, the Copilot system will identify and base on the validity of your student card to determine the limit for the Student development pack for Copilot. And of course, after the trial period ends, you will no longer be able to access Copilot for free, but you will have to pay to continue using it.

**Comment by AchrafEAE** [comment]
So I will not have copilot even though I am a student?

**Reply by t1noo7** [reply]
> So I will not have copilot even though I am a student?

That's correct, but first you have to do some step according its requirement to verify with the system that you are a student, otherwise the system will consider you as a normal trial user

**Comment by davevad93** [comment, marked as answer]
Hi @AchrafEAE ,

If you applied for the `Student Developer Pack` and your benefits are available, you should be able to still use `Copilot` for free after the trial period is over, as it comes with the pack. As for the duration, student benefits last for 2 years, after which you have to re-verify your student status.

If at the end of the trial period you can't use use `Copilot`, try to redeem your offer [here](https://education.github.com/pack/redeem/copilot-student), in case you can't redeem it, contact support by opening a ticket [here](https://support.github.com/contact/education), select “`I am a verified student having problems redeeming my Global Campus offers`” and explain your issue.

**Reply by davevad93** [reply]
However, remember that your benefits will be available when the status label is purple and says "`Congrats! Your request for @AchrafEAE  was approved and your benefits are now available.`"

**Reply by AchrafEAE** [reply]
Okay I guess I'll wait it out and see what happens. Thanks.

**Reply by AchrafEAE** [reply]
Yes it says my benefits are available.

**Reply by davevad93** [reply]
Ok @AchrafEAE , so just remember what I wrote above, if you have any problem try first to redeem your offer, if you can't do it, contact support by selecting the proper dropdown option "`I am a verified student having problems redeeming my Global Campus offers`" and they will solve it for you.

**Reply by AchrafEAE** [reply]
I just got an email from Github that says this:

Dear GitHub customer,

As you qualify for free access to GitHub Copilot, we recently sent you a refund receipt for your subscription. No other steps are needed to receive your free access to GitHub Copilot!

Note: It may take up to 14 days for the refund to be processed by your payment provider.

So I should be fine.

**Reply by davevad93** [reply]
Yes @AchrafEAE , everything is fine! 👍

**Reply by farshidjon1** [reply]
perfect

**Comment by B3NKaYS** [comment]
Hello, my documents were approved about 10 days ago, but Copilot still wants a pro membership with free features. I have projects to complete, I would be very happy if the process is accelerated.
"Hi, B3NKaYS! You were last verified as a student on Feb 14, 2025. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits."
![Ekran görüntüsü 2025-02-24 094735](https://github.com/user-attachments/assets/57eca54f-4bcd-467a-a6a7-c720c4297782)
![Ekran görüntüsü 2025-02-24 094836](https://github.com/user-attachments/assets/6d6868f1-593e-47fc-8508-70caa40e8568)

**Reply by chapi1234** [reply]
Mine is similar; my GitHub account is verified as Pro, but Copilot isn't reflecting the upgrade.

**Comment by misterkillerman** [comment]
hello, i've had a similar issue where i was using the free trial version before realising i can use the education benefits. i just cancelled my free trial but i am unable to redeem my educational benefits copilot. when i tried to redeem it, i have this error message:
<img width="301" height="136" alt="image" src="https://github.com/user-attachments/assets/90451956-86fd-4376-915e-73bdd205a5b8" />

even though i have cancelled it, it says this:
<img width="944" height="374" alt="image" src="https://github.com/user-attachments/assets/c457e599-6633-4335-b7ca-0e58a13bd2c4" />

but i still am not able to use copilot, and all my previous sessions are gone.
